thamos/discover.py

```diff
@@ -175,15 +288,85 @@ def discover_base_image() -> typing.Optional[str]:
     return None
 
 
-def discover_all() -> typing.Dict[str, typing.Any]:
+def discover_openblas_version() -> Optional[Dict[str, str]]:
+    """Check for OpenBLAS version and return None if not found."""
+    result = run_command("rpm -qa | grep openblas", raise_on_error=False)
+
+    if result.return_code != 0:
+        _LOGGER.debug("No OpenBLAS version detected")
+        _LOGGER.debug("OpenBLAS not found in RPM packages")
+        return None
+
+    if result.stdout is not None:
+        for line in result.stdout.splitlines():
+            name, ver, rel, epoch, arch = _split_rpm_filename(line)
+            if name == "openblas":
+                _LOGGER.debug(f"OpenBLAS detected with version {ver}")
```

Comment:
Best practice with the debugging library is to:
```
_LOGGER.debug("OpenBLAS detected with version %s", ver)
```
Minor performance improvement since formatting only has to be done in debug mode and it also improves log aggregation in things like Sentry

Reply:
Thanks!